AngularApplication/src/styles.scss

```diff
@@ -12,7 +12,7 @@
 @import "./scss/comments-editor-styles";
 
 // 3RD PARTY
-@import "~angular-star-rating/assets/scss/star-rating";
+@import "~css-star-rating/scss/star-rating";
```

Comment:
@todorivanov make sure that the project is building successfully after these changes. I remember that there were some build-time issues with the scss compilation...

Reply:
Seems okay for now. Its building properly.